docs/project/library-servicing.md

```diff
@@ -4,37 +4,12 @@ This document provides the steps necessary after modifying a library in a servic
 
 ## Check if a package is generated
 
-If a library's source project sets `<IsPackable>true</IsPackable>` a package is generated. If the library's source project doesn't set `<IsPackable>true</IsPackable>`, the library likely is part of the shared framework (to check for that, look into the `NetCoreAppLibrary.props` file). If it is, then there is nothing that needs to be done here.
+If a library likely is part of the shared framework (to check for that, look into the `NetCoreAppLibrary.props` file), then there is nothing that needs to be done here.
+Otherwise, You'll need to set `<GeneratePackageOnBuild>true</GeneratePackageOnBuild>` in the source project becuase the packages are not generated by default in the servicing releases.
```

Comment:
```suggestion
If a library is packable (check for the `<IsPackable>true</IsPackable>` property) you'll need to set `<GeneratePackageOnBuild>true</GeneratePackageOnBuild>` in the source project. That is necessary as packages aren't generated by default in servicing releases.
```

Reply:
Starting with 6.0 we have packages which contain assets that are also redistributed inside the shared framework. We don't trim these out of our packages anymore (net6.0 assets).